### ISSUE TITLE
IPYTHON_AVAILABLE evaluates to false because of failed import from bokeh

### ISSUE THREAD

**Issue by timothydmorton** [ioam/parambokeh#16]
here:  https://github.com/ioam/parambokeh/blob/master/parambokeh/__init__.py#L16 

Looks like this is the exception being raised; doesn't seem to be the one intended to be caught.  Seems to have popped up with bokeh 0.12.10.

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-14-119ad22f1a00> in <module>()
----> 1 from parambokeh.comms import JupyterCommJS, JS_CALLBACK, notebook_show

~/.local/lib/python3.6/site-packages/parambokeh-0.2.1-py3.6.egg/parambokeh/comms.py in <module>()
      4 import traceback
      5 
----> 6 from bokeh.embed import notebook_div
      7 
      8 from IPython.display import publish_display_data

ImportError: cannot import name 'notebook_div'
```

**Comment by ceball** [comment]
Thanks for the report. Fixing this is part of #11.

**Comment by ceball** [comment]
This has been fixed in #17, but it's still a work-in-progress PR at the moment. Please feel free to mention anything else you know of/discover in #11. Thanks!

**Comment by ceball** [comment]
Closing as it's covered by #11.
